process/rating/blockSigningRater.go

```diff
@@ -127,19 +145,51 @@ func (bsr *BlockSigningRater) ComputeIncreaseProposer(shardId uint32, currentRat
 	return bsr.computeRating(ratingStep, currentRating)
 }
 
-// ComputeDecreaseProposer computes the new rating for the decreaseLeader
-func (bsr *BlockSigningRater) ComputeDecreaseProposer(shardId uint32, currentRating uint32) uint32 {
+// ComputeIncreaseProposer computes the new rating for the increaseLeader
+func (bsr *BlockSigningRater) RevertIncreaseValidator(shardId uint32, currentRating uint32, nrReverts uint32) uint32 {
```

Comment:
change comment - maybe add a better naming to the function.

Reply:
fixed